contracts/FeeSettings.sol

```diff
@@ -65,6 +88,17 @@ contract FeeSettings is Ownable2Step, ERC165, IFeeSettingsV2, IFeeSettingsV1 {
         uint32 privateOfferFeeDenominator
     );
 
+    event SetCustomFee(
+        address indexed token,
+        uint32 tokenFeeNumerator,
+        uint32 tokenFeeDenominator,
+        uint32 crowdinvestingFeeNumerator,
+        uint32 crowdinvestingFeeDenominator,
+        uint32 privateOfferFeeNumerator,
+        uint32 privateOfferFeeDenominator,
+        uint256 time
```

Comment:
expiration date is a better description than `time`

Reply:
I see. What about "ValidityDate"?

Reply:
ok, we can do validityDate

Reply:
done [fe6e452](https://github.com/corpus-io/tokenize.it-smart-contracts/pull/270/commits/fe6e4520dc6aa1b90d465eb6ffc84f9b8e6c50e2)